deps/build.jl

```diff
@@ -59,6 +59,7 @@ end
 
 # Windows
 if is_windows()
+    rm(joinpath(dirname(@__FILE__), "usr"); force = true, recursive = true)
```

Comment:
Alright. Is this warning okay: 
```
warn("Looks like your binary is old. Please run `rm(joinpath(Pkg.dir(\"HttpParser\"), \"deps\", \"usr\"; force = true, recursive = true)` to delete the old binary and then run `Pkg.build(\"HttpParser\")` again.")
```
( I ask because you've advised against using `Pkg.dir` and `@__FILE__` doesn't work on the REPL.)